cli/lib/kontena_cli.rb

```diff
@@ -87,6 +115,26 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    if log_target.respond_to?(:tty?) && log_target.tty?
+      logger = Logger.new(log_target)
+      logger.datetime_format = "%M%S.%2N "
```

Comment:
That's how I would read it. So clearly needs some tuning still. I'd include also the hour, if a command runs just when the hour changes, the output would jump from `5959.90` -> `0000.10` or something alike. Just not readable/understandable IMO.